gazebo_color_plugin/worlds/light.world

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
+ <world name="default">
+ <include>
+   <uri>model://ground_plane</uri>
+ </include>
+
+ <include>
+   <uri>model://sun</uri>
+ </include>
+
+ <!-- reference to your plugin -->
+ <model name='camera'>
+   <pose>0 -1 0.05 0 -0 0</pose>
+   <link name='link'>
+     <inertial>
+       <mass>0.1</mass>
+       <inertia>
+        <ixx>1</ixx>
+         <ixy>0</ixy>
+         <ixz>0</ixz>
+         <iyy>1</iyy>
+         <iyz>0</iyz>
+         <izz>1</izz>
+       </inertia>
+     </inertial>
+     <collision name='collision'>
+       <geometry>
+         <box>
+            <size>0.1 0.1 0.1</size>
+         </box>
+       </geometry>
+       <max_contacts>10</max_contacts>
+       <surface>
+         <contact>
+           <ode/>
+         </contact>
+         <bounce/>
+         <friction>
+           <ode/>
+         </friction>
+       </surface>
+     </collision>
+     <visual name='visual'>
+       <geometry>
+         <box>
+           <size>0.1 0.1 0.1</size>
+         </box>
+       </geometry>
+     </visual>
+     <sensor name='camera' type='camera'>
+       <camera name='__default__'>
+         <horizontal_fov>1.047</horizontal_fov>
+         <image>
+           <width>320</width>
+           <height>240</height>
+         </image>
+         <clip>
+           <near>0.1</near>
+           <far>100</far>
```

Comment:
Is it necessary to include such a far clip distance?

Reply:
File unnecessary, deleted